front_end/helper.py

```diff
@@ -56,27 +57,48 @@ def is_old_file(file_path, days=30):
 
 def convert_markdown_to_html(text):
     html = text
```

Comment:
This is probably something I left in there, but please rename these variables. You can get rid of this line (html = text) and rename the variable you use with the markdown2 module to html.